benchmark/Streamly/Benchmark/Data/Array/Unboxed/Mut.hs

```diff
@@ -78,9 +77,6 @@ benchIOSink value name = benchIO' name (sourceIntFromTo value)
 benchIOSrc :: String -> (Int -> IO (Stream a)) -> Benchmark
 benchIOSrc name src = benchIO name src id
 
-drain :: Monad m => Stream.Stream m a -> m ()
-drain = Stream.fold Fold.drain
-
```

Comment:
This change could have been kept. It would have been useful to remove Streamly.Prelude from this module in future. We should import only minimal stuff needed from Streamly.Prelude.